src/rdkafka_topic.c

```diff
@@ -377,6 +377,21 @@ rd_kafka_topic_t *rd_kafka_topic_new0 (rd_kafka_t *rk,
                 }
         }
 
+        if (rkt->rkt_rk->rk_conf.sticky_partition_linger_ms > 0 &&
+            rkt->rkt_conf.partitioner !=
```

Comment:
What happens with custom partitioners set by the application?

Reply:
If a custom partitioner is set by the application, non-keyed messages will still undergo sticky partitioning unless sticky partitioning is disabled.